dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportController.java

```diff
@@ -200,7 +200,10 @@ ResponseEntity<ObjectNode> getTrackedEntityByUid(
     TrackedEntity trackedEntity =
         TRACKED_ENTITY_MAPPER.from(
             trackedEntityService.getTrackedEntity(
-                uid.getValue(), program == null ? null : program.getValue(), trackedEntityParams));
+                uid.getValue(),
+                program == null ? null : program.getValue(),
+                trackedEntityParams,
+                false));
```

Comment:
Can you please make sure we have a test in TrackedEntitiesExportControllerTest that tests users get a 404 if they request a soft-deleted trackedEntity 😄